src/containers/StoryList.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router";
+
+import { getStoryIds } from "../services/hnAPI";
+
+import { Story } from "../components/Story";
+
+import { StoryListWrapper, ButtonLink } from "../styles/GlobalStyle";
+import { LINKS_PER_PAGE, MAX_PAGES} from "../constants";
+
+
+export default function StoryList({ history }) {
+  const [storyIds, setStoryIds] = useState([]);
+  let { page } = useParams();
+
+  // for pagination
+  let startStory = LINKS_PER_PAGE * +page,
+    endStory = LINKS_PER_PAGE + LINKS_PER_PAGE * +page - 1;
```

Comment:
Since this is broken into two lines anyway, I would just use two `let` statements here

Reply:
I'm also not a huge fan of how `endStory` is calculated. I think the following is easier to read, for example:

```js
let endStory = startStory + LINKS_PER_PAGE - 1
```

It's a nice touch to see the unary plus operator here, but I don't think it's needed. `useParams` should make sure that you only get numbers back in the first place.

Reply:
updated nice comments, I'm already learning a lot from you.